src/components/ButtonGroup/Buttons.tsx

```diff
@@ -0,0 +1,40 @@
+import React, {useState} from 'react';
+import {useDispatch} from "react-redux";
+import {sortTickets, sortTicketsRevert} from "../../store/reducers/ticketsReducer";
+
+
+const Buttons = () => {
+    const [value, setValue] = useState('1');
+
+    let dispatch = useDispatch()
+
+    return (
+
+        <div className={'ml-[20px]'}>
+            <button
+                type={"button"}
+                className={'border-solid border-2 border-gray-400 rounded-l-lg bg-white hover:bg-blue-50 w-[168px] h-[50px]'}
+                onClick={() => dispatch(sortTickets())}
+            >
+                Самый дешевый
+            </button>
+            <button
+                type={"button"}
+                className={'border-solid border-2 border-gray-400 bg-white  hover:bg-blue-50 w-[168px] h-[50px]'}
+                onClick={() => dispatch(sortTicketsRevert())}
+            >
+                Самый быстрый
+            </button>
+            <button
+                type={"button"}
+                className={'border-solid border-2 border-gray-400 rounded-r-lg bg-white hover:bg-blue-50 w-[168px] h-[50px]'}
+            >
+                Оптимальный
+            </button>
```

Comment:
Дубляж кода. Лучше было бы создать отдельную переменную, скажем 

```
const buttons = [
  {id: 1, label: 'Самый дешевый', action: () => dispatch(sortTickets())},
  {id: 2, label: 'Самый быстрый', action: () => dispatch(sortTicketsRevert())},
  {id: 3, label: 'Оптимальный', action: () => {}},
]
```

```suggestion
          {buttons.map((button) => {
            <button
              key={button.id}
              type={"button"}
              className={'border-solid border-2 border-gray-400 rounded-l-lg bg-white hover:bg-blue-50 w-[168px] h-[50px]'}
              onClick={button.action}
            >
              {button.label}
            </button>
          })};

```